ServiceWebsite/ServiceWebsite.AcceptanceTests/TestClients/ApiTestClientBase.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using RestSharp;
+using ServiceWebsite.Common;
+
+namespace ServiceWebsite.AcceptanceTests.TestClients
+{
+    public class ApiTestClientBase : IApiClient
+    {
+        protected RestClient _client;
```

Comment:
As thisis protected, it is available in derived classes anyway, so no need to have the Client property

Reply:
done